### ISSUE TITLE
run_mass_enzh.sh cannot be stopped and restarted from checkpoint

### ISSUE THREAD

**Issue by moyid** [microsoft/MASS#84]
I'm running this pretraining script, and have obtained some good results. However, I've found that, if I stop the training, run some evaluations or what not, and restart it, it reads the latest checkpoint fine, but the future checkpoints are gibberish -- actually it looks like the future checkpoint are no longer translating Chinese into English, but the other way -- it's attempting to translate English into Chinese but producing very poor results. BLEU scores on test set are generally around 0. However, it still does ok on the validation set I'm guessing, because the best checkpoints are usually stored as one of these gibberish checkpoints.
